README.md

```diff
@@ -0,0 +1,19 @@
+CHT Pipeline
+========================
+
+CHT Pipeline is a tool used to define data models for transforming the raw data we get from Couch DB into models that can then be queried to build dashboards
+
+# Installation
+This project canbe run in one of two ways:
+1. Using the docker image and passed as an argument to [cht-sync](https://github.com/medic/cht-sync)
+2. Using the local installation of DBT
+
+## Docker
+1. Follow the instructions in [cht-sync](https://github.com/medic/cht-sync) to set up the required environment variables
+1. Add the path to your pipline banch to the [docker-compose.postgres.yml](https://github.com/medic/cht-sync/blob/main/docker-compose.postgres.yml#L13) file in the cht-sync project
```

Comment:
Change this to be an environment variable. If you edit the docker-compose source directly then checking out later versions may conflict or you might accidentally commit it back to the main branch.